concepticondata/conceptlists/Marchese-1983-311.tsv

```diff
@@ -0,0 +1,312 @@
+Marchese-1983-311-34	34	pintade	456	GUINEA FOWL	guineafowl	LES ANIMAUX
+Marchese-1983-311-35	35	poisson	227	FISH	fish	LES ANIMAUX
+Marchese-1983-311-36	36	pou	1392	LOUSE	louse	LES ANIMAUX
+Marchese-1983-311-37	37	poule	1514	HEN	hen	LES ANIMAUX
+Marchese-1983-311-38	38	python (boa)			python (boa)	LES ANIMAUX
+Marchese-1983-311-39	39	rat	1490	RAT	rat	LES ANIMAUX
+Marchese-1983-311-40	40	sauterelle	607	GRASSHOPPER	grasshopper	LES ANIMAUX
+Marchese-1983-311-41	41	scorpion	1538	SCORPION	scorpion	LES ANIMAUX
+Marchese-1983-311-42	42	serpent	730	SNAKE	snake	LES ANIMAUX
+Marchese-1983-311-43	43	silure			catfish	LES ANIMAUX
+Marchese-1983-311-44	44	souris	1491	MOUSE	mouse	LES ANIMAUX
+Marchese-1983-311-45	45	termite	883	TERMITE	termite	LES ANIMAUX
+Marchese-1983-311-46	46	tortue (terre)	895	TORTOISE	turtle (ground)	LES ANIMAUX
+Marchese-1983-311-47	47	ver	1219	WORM	worm	LES ANIMAUX
+Marchese-1983-311-48	48	arbre	906	TREE	tree	LES VEGETAUX
+Marchese-1983-311-49	49	banane plantain	399	PLANTAIN	plantain	LES VEGETAUX
+Marchese-1983-311-50	50	bois de chauffage	10	FIREWOOD	firewood	LES VEGETAUX
+Marchese-1983-311-51	51	brousse	1683	BUSH	bush	LES VEGETAUX
+Marchese-1983-311-52	52	caouchouc			caoutchouc	LES VEGETAUX
+Marchese-1983-311-53	53	champignon	641	MUSHROOM	mushroom	LES VEGETAUX
+Marchese-1983-311-54	54	charbon de bois	1188	CHARCOAL	charcoal	LES VEGETAUX
+Marchese-1983-311-55	55	citron	3091	LEMON	lemon	LES VEGETAUX
+Marchese-1983-311-56	56	coco	970	COCONUT	coconut	LES VEGETAUX
+Marchese-1983-311-57	57	éponge			sponge	LES VEGETAUX
+Marchese-1983-311-58	58	feu	221	FIRE	fire	LES VEGETAUX
+Marchese-1983-311-59	59	fromager			cheesemonger	LES VEGETAUX
+Marchese-1983-311-60	60	fruit	1507	FRUIT	fruit	LES VEGETAUX
+Marchese-1983-311-61	61	gombo	1737	OKRA	okra	LES VEGETAUX
+Marchese-1983-311-62	62	graine	714	SEED	seed	LES VEGETAUX
+Marchese-1983-311-63	63	graine de palme			palm seed	LES VEGETAUX
+Marchese-1983-311-64	64	herbe	606	GRASS	grass	LES VEGETAUX
+Marchese-1983-311-65	65	huile	1484	OIL (ORGANIC SUBSTANCE)	oil	LES VEGETAUX
+Marchese-1983-311-66	66	igname	410	YAM	yam	LES VEGETAUX
+Marchese-1983-311-67	67	kaolin			kaolin	LES VEGETAUX
+Marchese-1983-311-68	68	kola			kola	LES VEGETAUX
+Marchese-1983-311-69	69	maïs	506	MAIZE	but	LES VEGETAUX
+Marchese-1983-311-70	70	mil	931	MILLET	mil	LES VEGETAUX
+Marchese-1983-311-71	71	palmier à huile			oil palm tree	LES VEGETAUX
+Marchese-1983-311-72	72	palmier (raphia)	1181	PALM TREE	palm (raffia)	LES VEGETAUX
+Marchese-1983-311-73	73	papaye	2445	PAPAYA	papaya	LES VEGETAUX
+Marchese-1983-311-74	74	piment	919	CHILI PEPPER	chili pepper	LES VEGETAUX
+Marchese-1983-311-75	75	riz	926	RICE	rice	LES VEGETAUX
+Marchese-1983-311-76	76	tabac	974	TOBACCO	tobacco	LES VEGETAUX
+Marchese-1983-311-77	77	taro	351	TARO	taro	LES VEGETAUX
+Marchese-1983-311-78	78	albinos			albino	L'ÊTRE HUMAIN
+Marchese-1983-311-79	79	ami	1325	FRIEND	friend	L'ÊTRE HUMAIN
+Marchese-1983-311-80	80	bété			stupid	L'ÊTRE HUMAIN
```

Comment:
Could be mapped to 1518 STUPID, although the form normally looks like this: bête
I don't know what bété means, a quick search revealed nothing.

Reply:
I also think this is a spelling error of the French form, maybe this happened when typing, or during OCR, if the English form "stupid" is also given in the source?

Reply:
I also thought so at first but it's in fact a language name (it's listed in the target languages in the source of the list).

Reply:
Okay, then the translation should be modified.